README.md

```diff
@@ -1,18 +1,25 @@
 ![npm](https://img.shields.io/npm/v/@asyncapi/specs?style=for-the-badge) ![npm](https://img.shields.io/npm/dt/@asyncapi/specs?style=for-the-badge)
+
 > If you are currently using version 2, check out [migration guideline to version 3](./migrations/Migrate%20to%20version%203.md). You might be able to update it without any change.
```

Comment:
we need to update this part a bit. 
we talk about migration from 2 ->3 but then there is v4, so might be confusing

maybe we need dedicated section, like `Migrations`, where we list path from 2 -> 3 and explain that 3 -> 4 means that we no longer support versions prior 2.0 which means migration path is to switch to 2.x version of the spec